### PR TITLE
Origin/setting/workspaceprofile/update workspace name working

### PR DESCRIPTION
**Pull request by darshanitaliya7110**
### What this does
update workspace name working

### Implementation
create updateWorkspaceDetails function and use it for update organizationName name 

### Testing
https://share.teamcamp.app/D6AOa2SdTLLnq3ZIDpGlE

### CODE REVIEW

File: app/setting/workspaceprofile/page.tsx [hunk 1]
```diff
@@ -20,8 +20,23 @@ import {
...
+  const { workspaceStore } = useStores();
+  const { currentWorkspace } = workspaceStore;
+  const [organizationName, setOrganizationName] = useState(
+    currentWorkspace?.name,
+  );
+
+  const handleUpdate = (e: SyntheticEvent) => {
+    e.preventDefault();
+    if (currentWorkspace)
+      workspaceStore.setCurrentWorkspace({ ...currentWorkspace });
```
Comment (aniket-pixer) [inline app/setting/workspaceprofile/page.tsx:36]: workspaceStore.setCurrentWorkspace({ ...(currentWorkspace || {}), name: organizationName});

File: app/setting/workspaceprofile/page.tsx [hunk 2]
```diff
@@ -20,8 +20,23 @@ import {
...
+    if (organizationName) updateWorkspaceDetails(organizationName);
```
Comment (aniket-pixer) [inline app/setting/workspaceprofile/page.tsx:37]: if (organizationName) updateWorkspaceDetails(organizationName);

when we call service for any form then we will create const for payload and pass that
const payload = { name: organizationName }
updateWorkspaceDetails(payload);

File: services/clientSide/workspaceServices.ts [hunk 3]
```diff
@@ -106,3 +106,25 @@ export const getWorkspaceList = async () => {
     workspaceStore.setLoading(false);
   }
 };
+
+/**
+ * @desc Update workspace details
+ * @param {*} organizationName
+ */
+export const updateWorkspaceDetails = async (organizationName: string) => {
```
Comment (aniket-pixer) [inline services/clientSide/workspaceServices.ts:114]: In parameter we will recieve payload so update based on that

File: app/setting/style.tsx [hunk 4]
```diff
@@ -123,7 +123,7 @@ export const StyledCheckbox = styled.input.attrs({ type: 'checkbox' })`
 
   &:checked {
     background-color: ${colors.brand};
-    background-image: url('data:image/svg+xml;utf8,${encodeURIComponent(checkmarkSVG)}');
+    background-image: url('data:image/svg+xml;utf8,${encodeURIComponent(checkmarkSVG,)}');
```
Comment (aniket-pixer) [inline app/setting/style.tsx:126]: Remove ,

File: app/setting/workspaceprofile/page.tsx (hunk 1 above)
Comment (darshanitaliya7110) [reply app/setting/workspaceprofile/page.tsx]: Done

File: app/setting/workspaceprofile/page.tsx (hunk 2 above)
Comment (darshanitaliya7110) [reply app/setting/workspaceprofile/page.tsx]: Done

**Review by aniket-pixer — APPROVED** [review]

**Pull request merged**